spring/src/test/java/io/kubernetes/client/spring/extended/controller/KubernetesReconcilerCreatorTest.java

```diff
@@ -94,7 +94,7 @@ public SharedInformerFactory sharedInformerFactory() {
                   apiVersion = "v1",
                   resourcePlural = "configmaps")),
     })
-    static class TestSharedInformerFactory extends SharedInformerFactory {}
+    static class TestSharedInformerFactory {}
```

Comment:
IMO the naming might be more close to the semantic. It's just a "holder" for @KubernetesInformers, kind of configuration.

Reply:
Just lazy here. The name was from the old test.